cli/client.go

```diff
@@ -62,163 +37,6 @@ func NewClient(baseURL url.URL) (Client, error) {
 	return Client{http: httpClient, baseURL: baseURL}, nil
 }
 
-func (c *Client) do(ctx context.Context, path string, r io.Reader, responseStruct interface{}) error {
```

Comment:
This was removed so that we don't accidentally use these.

These API calls are no longer used in this version of KeyConjurer.

The API will be kept within the repository until such a time that we know all users have migrated away from the existing version of KeyConjurer. We can use our metrics for that but this will largely be a announcement-driven change, I expect.